PROJECT_CHARTER.md

```diff
@@ -0,0 +1,51 @@
+# `jest` Charter
+
+Jest is a modular test runner and framework for JavaScript, designed for running using Node.js on all of its supported platforms. It is a modular runner than can run and test any JavaScript, TypeScript, or other language that compiles to JavaScript. It is designed to be easy to use and to have a low barrier to entry, while still being flexible and powerful.
+
+## Section 0: Guiding Principles
+
+The `jest` project is part of the [OpenJS Foundation][openjs foundation], which operates transparently, openly, collaboratively, and ethically. Project proposals, timelines, and status must not merely be open, but also easily visible to outsiders.
+
+## Section 1: Scope
+
+`jest` is primarily a test framework and test runner, focused on making it easy to test any JS code, either written for the server, the client or both. The Jest project consists of many different smaller modules, such as the assertion library, test runner, diffing algorithm, object serializer, task manager and more. The project is designed to be modular, so that it can be used in a variety of different environments and for different purposes.
+
+## Section 2: Relationship with OpenJS Foundation CPC.
+
+Technical leadership for the projects within the [OpenJS Foundation][openjs foundation] is delegated to the projects through their project charters by the [OpenJS Foundation Cross-Project Council](https://openjsf.org/about/governance/) (CPC). In the case of the `jest` project, it is delegated to the [`jest` Maintainers](GOVERNANCE.md#maintainers) (the “Maintainers”). The OpenJS Foundation's business leadership is the Board of Directors (the “Board”).
+
+This `jest` Charter reflects a carefully constructed balanced role for the Maintainers and the CPC in the governance of the OpenJS Foundation. The charter amendment process is for the Maintainers to propose changes using simple majority of the full Maintainers, the proposed changes being subject to review and approval by the CPC. The CPC may additionally make amendments to the project charter at any time, though the CPC will not interfere with day-to-day discussions, votes or meetings of the Maintainers.
+
+### 2.1 Other Formal Project Relationships
+
+Section Intentionally Left Blank
+
+## Section 3: `jest`'s Maintainers Governing Body
+
+`jest` is governed by its [maintainers](GOVERNANCE.md#maintainers).
+
+## Section 4: Roles & Responsibilities
+
+The roles and responsibilities of `jest`'s Maintainers are described in [GOVERNANCE.md](./GOVERNANCE.md).
+
+### Section 4.1 Project Operations & Management
+
+Section Intentionally Left Blank
+
+### Section 4.2: Decision-making, Voting, and/or Elections
+
+Section Intentionally Left Blank
```

Comment:
It would be nice to state who makes project decisions and how (is it by consensus, voting, etc.)

Reply:
Since I'm the only active maintainer, I don't know what to put here that's. I'd love for there to be more stakeholders/maintainers involved in the process (and in maintaining the project as a whole), but at the moment any voting/consensus would just be myself 😅

Reply:
I think you being candid about it and being a little aspirational here would make total sense, e.g.

> When the project was onboarded, it only had one maintainer. The goal is to grow the pool of maintainers and to collectively define a consensus-driven governance model that will be defined in GOVERNANCE.md.

or:

> The project will define a consensus-driven governance model in GOVERNANCE.md as need arises.

Reply:
I think Tobie's second suggestion will work well and provides future flexibility.

Reply:
[`096202f` (#14948)](https://github.com/jestjs/jest/pull/14948/commits/096202f7eb43fdfe9e610b68d1a79e8435f4aa1a)